mkdocs.yml

```diff
@@ -560,6 +605,7 @@ nav:
         - Data:
             - Overview:                                 properties/data/overview.md
             - Core:                                     properties/data/core.md
+              # Add example schema for new file_content property
```

Comment:
We should explain all IO unit types then

Reply:
Updated to clarify this